front/src/services/auth.ts

```diff
@@ -0,0 +1,65 @@
+import { auth } from "./firebase";
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword, signOut, updateProfile } from 'firebase/auth';
+import { FirebaseError } from "../@types/auth";
+
+const getErrors = (code: string) => {
```

Comment:
Je suis pas super fan du switch case à ralonge, je verrais plutot un gros map 
`const errorMapping = {'auth/email-already-exists':'The provided email is already in use by an existing user. Each user must have a unique email.'}`

Reply:
C'est plus lisible pour de grosse structure conditionelle et en bas t'a juste a passé la clé et il va te retourner la string :)

Reply:
ok je pense qu'on va le faire dans le common avec le mobile aussi